src/DepthImageToLaserScanROS.cpp

```diff
@@ -47,36 +47,29 @@
 using namespace depthimage_to_laserscan;
 
 DepthImageToLaserScanROS::DepthImageToLaserScanROS(rclcpp::Node::SharedPtr & node):node_(node) {
-
-  cam_info_sub_ = node_->create_subscription<sensor_msgs::msg::CameraInfo>("depth_camera_info",
+  auto qos = rclcpp::QoS(rclcpp::KeepLast(10));
+  cam_info_sub_ = node_->create_subscription<sensor_msgs::msg::CameraInfo>("depth_camera_info", qos,
       std::bind(
         &DepthImageToLaserScanROS::infoCb, this,
-        std::placeholders::_1),
-      rmw_qos_profile_sensor_data);
+        std::placeholders::_1));
 
   depth_image_sub_ =
-    node_->create_subscription<sensor_msgs::msg::Image>("depth",
-      std::bind(&DepthImageToLaserScanROS::depthCb, this, std::placeholders::_1),
-      rmw_qos_profile_sensor_data);
+    node_->create_subscription<sensor_msgs::msg::Image>("depth", qos,
+      std::bind(&DepthImageToLaserScanROS::depthCb, this, std::placeholders::_1));
 
-  scan_pub_ = node_->create_publisher<sensor_msgs::msg::LaserScan>("scan");
+  scan_pub_ = node_->create_publisher<sensor_msgs::msg::LaserScan>("scan", qos);
 
-  float scan_time = 0.033;
-  node_->get_parameter("scan_time", scan_time);
+  float scan_time = node_->declare_parameter("scan_time", 0.033);
   dtl_.set_scan_time(scan_time);
 
-  float range_min = 0.45;
-  float range_max = 10.0;
-  node_->get_parameter("range_min", range_min);
-  node_->get_parameter("range_max", range_max);
+  float range_min = node_->declare_parameter("range_min", 0.15);
```

Comment:
This used to be a default of 0.45 and 10.0, respectively.  Why are we changing the defaults?

Reply:
My mistake:), I  changed this value before according to my demand during developing and forget to revert it back